js/index.js

```diff
@@ -0,0 +1,21 @@
+  const mainContainer = makeElement('div', { className: 'main-container' });
+  const formContainer = makeElement('div', { className: 'form-container' });
+  const resultsContainer = makeElement('div', {
+    className: 'results-container',
+  });
+  
+  function createMain() {
+    mainContainer.append(formContainer, resultsContainer);
+    document.body.append(mainContainer);
+  }
+
+  window.onload = async function () {
+    const marContainer = document.querySelector('.contentMarq');
+    const marquee = new Marquee(marContainer);
+    marquee.start();
+  
+    const resultsPanel = new ResultsPanel(resultsContainer);
+    const search = new Search(formContainer, resultsPanel);
+    createMain();
```

Comment:
your onload function is a little different from what we asked. There is a good reason to bind the classes instances like showed in the instarctions. Please ask your lecturer/mentor about this issue.